exercises/practice/hamming/.docs/instructions.append.md

```diff
@@ -0,0 +1,14 @@
+# Implementation notes: exceptions
+
+This exercise expects the code to raise exceptions in certain circumstances.
+Exceptions are usually raised when an operation cannot be completed for some
+reason and there is no value that can be returned which would be correct.
+Instead, an exception is raised. When raising an exception, the tests expect
```

Comment:
Doesn't seem to be an "instead" clause as the previous sentence already says there is no clause. `Therefore` might be more applicable.

Reply:
Changed.